package.json

```diff
@@ -33,17 +36,24 @@
     "enzyme-adapter-react-16": "^1.12.1",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
+    "i": "^0.3.6",
     "moment": "^2.24.0",
+    "moxios": "^0.4.0",
     "node-sass": "^4.12.0",
-    "prop-types": "^15.7.2",
+    "npm": "^6.9.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-facebook-login": "^4.1.1",
+    "react-google-login": "^5.0.4",
+    "prop-types": "^15.7.2",
     "react-notify-toast": "^0.5.0",
     "react-redux": "^7.0.3",
     "react-router-dom": "^5.0.0",
     "react-toolbox": "^2.0.0-beta.13",
+    "react-twitter-auth": "0.0.13",
```

Comment:
You can remove this since you don't have twitter configured yet

Reply:
@adesege worked on feedback